src/main/java/pl/h2security/controller/HomeController.java

```diff
@@ -0,0 +1,23 @@
+package pl.h2security.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class HomeController {
+
+    @RequestMapping("/")
+    @ResponseBody
+    public String home(Model model) {
+        return "Coś tam coś tam";
+    }
+
+
+    @RequestMapping("/dupa")
```

Comment:
Remove this endpoint and as I said before, don't add things like that to public repo's. It is unprofessional.